parsers/csv/csv.go

```diff
@@ -17,10 +17,11 @@ import (
 // Options defines the options relevant to the CSV parser
 type Options struct {
 	Fields           string `long:"fields" description:"Comma separated list of CSV fields, in order."`
```

Comment:
Should we add a `yam:"-"` for Fields too?

Reply:
Definitely not "-" -- that would prevent the field from ever showing up. The rule I used was to add a YAML declaration only when the field name used in config didn't match the variable name, or when it should be suppressed. In this case, the field is called "Fields" and that's also the config value name, so a YAML tag naming the field would be redundant.